packages/extensions/venia-virtual-products/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "@magento/venia-virtual-products",
```

If this implementation is not going to be the final core implementation it would make sense to follow the sample naming convention and not use up the logical name.
```suggestion
  "name": "@magento/venia-sample-virtual-products",
```

---

Good call out, I've gone ahead and made this change since this is really for reference given our specific sample data.